corehq/apps/app_manager/static/app_manager/js/case-config-ui.js

```diff
@@ -11,9 +11,8 @@ hqDefine('app_manager/js/case-config-ui.js', function () {
         var self = this;
         var i;
         self.help_name = gettext('Case Property Description');
-        self.default_description = gettext("You do not have a description for this case property. If you would like to add one you can do so in the data dictionary.");
         self.makePopover = function () {
-            $('.property-description').popover({
+            $('.read-only').popover({
```

Comment:
Super picky, but it'd be better to leave this as `.property-description`; this grabs every inline edit widget on the page (including the form title & comment).